draco/analysis/transform.py

```diff
@@ -258,8 +258,10 @@ def pack_product_array(arr):
         # Create output container
         if isinstance(ss, containers.SiderealStream):
             OutputContainer = containers.SiderealStream
+            output_kwargs = {"ra": ss.ra[:]}
```

Comment:
This one presumably isn't strictly necessary? I think the RA axis is always pixel centered.

Reply:
Probably safe to leave it in there though.